Resources/app/StoreFront/test/e2e/cypress/integration/extended-block.spec.js

```diff
@@ -0,0 +1,9 @@
+// / <reference types="Cypress" />
+describe('PluginExtendJsPlugin: Test is extended', () => {
+
+    it('checks for correct alert text', () => {
+        cy.visit('/');
+
+        cy.contains('Hello world!').should('be.visible');
```

Comment:
shouldn't we check for the correct div element or something similar? or is this not important?